solr/core/src/test/org/apache/solr/util/TestCircuitBreaker.java

```diff
@@ -114,10 +108,10 @@ public void testCBFakeMemoryPressure() {
 
     removeAllExistingCircuitBreakers();
 
-    PluginInfo pluginInfo = h.getCore().getSolrConfig().getPluginInfo(CircuitBreakerManager.class.getName());
+    CircuitBreaker circuitBreaker = new FakeMemoryPressureCircuitBreaker();
+    MemoryCircuitBreaker memoryCircuitBreaker = (MemoryCircuitBreaker) circuitBreaker;
```

Comment:
Declare circuitBreaker as a MemoryCircuitBreaker so we don't need this cast. Same pattern below too.